packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -225,5 +257,9 @@ const messagesWithContentOrFileSharingMetadata = (message: ChatMessageWithStatus
   if (message.metadata?.['fileSharingMetadata']) {
```

Comment:
Also a fun idea to explore - currently we are not showing deleted message (if you look at deletedOn filter on the top of this function) - is it possible to make DLP message as a special type deleted message(we don't have deleted message type as well), or is there a common path/pattern we can reuse DLP code for deleted message?

Reply:
Which means we can set messageType = 'deleted' but add a flag to say it is for dlp reason

Reply:
Interesting idea, but technically DLP message is still a message, and it can be deleted and based on the current logic it will be completely not shown when someone delete's their DLP message. Just like a regular ChatMessage or edited ChatMessage they can be deleted and with current logic if `onDelete` is not undefined then it will be filtered out.

Reply:
Yep, this is the current logic, while the ideal logic is when someone deleted the message the message will be shown as "this message has been deleted". Then it is pretty much like DLP and we might be able to reuse similar code even same type structure - but again, this is just an exploration we could do, not required for this PR